packages/create-next-app/templates/default/README-template.md

```diff
@@ -14,6 +14,10 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
 
+Demo the API server by opening [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+
+With Next.js, the `pages/api` directory is mapped to `/api/*` endpoints. Files in this directory are treated as API endpoints instead of pages.
```

Comment:
```suggestion
The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
```